Scripts/Python/xYeeshaPages.py

```diff
@@ -166,22 +167,24 @@ def OnGUINotify(self,id,control,event):
                 
                 PtDebugPrint ("xYeeshaPages.py: The previous value of the SDL variable %s is %s" % ("YeeshaPage" + str(PageNumber.value), YeeshaPageVar.getInt()))
 
+                if StartInCleft() and not IsCleftSolved():
```

Comment:
```suggestion
                if StartInCleft():
```

`StartInCleft()` implies `not CleftSolved()`.